Index: libxcb-1.1/src/xcb_conn.c
===================================================================
--- libxcb-1.1.orig/src/xcb_conn.c	2008-03-10 12:02:22.000000000 +0200
+++ libxcb-1.1/src/xcb_conn.c	2008-03-10 12:10:21.000000000 +0200
@@ -62,8 +62,10 @@
 static int _xcb_xlib_init(_xcb_xlib *xlib)
 {
     xlib->lock = 0;
+    xlib->sloppy_lock = 1;
 #ifndef NDEBUG
-    xlib->sloppy_lock = (getenv("LIBXCB_ALLOW_SLOPPY_LOCK") != 0);
+    if (getenv("LIBXCB_DISABLE_SLOPPY_LOCK"))
+        xlib->sloppy_lock = 0;
 #endif
     pthread_cond_init(&xlib->cond, 0);
     return 1;
