diff -ur libxcb-1.14.orig/configure.ac libxcb-1.14/configure.ac
--- libxcb-1.14.orig/configure.ac	2020-02-22 20:50:14.000000000 +0000
+++ libxcb-1.14/configure.ac	2022-04-05 12:56:51.059604090 +0000
@@ -12,7 +12,7 @@
 
 # Initialize Automake
 AM_INIT_AUTOMAKE([foreign dist-xz])
-AM_PATH_PYTHON([2.6])
+AM_PATH_PYTHON([3])
 
 # Set common system defines for POSIX extensions, such as _GNU_SOURCE
 # Must be called before any macros that run the compiler (like AC_PROG_LIBTOOL)
